cvat/apps/engine/models.py

```diff
@@ -275,7 +275,7 @@ def get_available_name(self, name, max_length=None):
         return name
 
 def upload_path_handler(instance, filename):
-    return os.path.join(instance.data.get_upload_dirname(), filename)
+    return os.path.join(os.path.relpath(instance.data.get_upload_dirname(), settings.BASE_DIR), filename)
```

Comment:
@MashaSS , Please add a comment why we have to do the change.

Reply:
Added a comment to function and a line to CHANGELOG.md.